src/Compute/Compute/ChangeLog.md

```diff
@@ -21,6 +21,7 @@
 -->
 ## Upcoming Release
 * Fixed `Get-AzVm` cmdlet when parameter "-Status" is provided, return property `OsName`, `OsVersion` and `HyperVGeneration`
+* Fixed 'New-AzVM' cmdlet when creating VM with bootdiagnostic storage causes exception 'Kind" cannot be null.
```

Comment:
Please use ` instead of ' to quote New-AzVM and please correct 'Kind"

Reply:
Corrected